rrset.go

```diff
@@ -210,12 +210,15 @@ type SBRDataInfo struct {
 }
 
 // BackupRecord wraps the backupRecord objects of an SBPoolProfile response
+// AvailableToServe required by UltraDNS restapi
 type BackupRecord struct {
 	RData         string `json:"rdata,omitempty"`
 	FailoverDelay int    `json:"failoverDelay,omitempty"`
+        AvailableToServe bool   `json:"availableToServe,omitempty"`
```

Comment:
Can we run this file through `go fmt`? This block is oddly indented now.

Your comment mentions this is required, should we skip `omitempty`?

Reply:
Yup sure will edit using go fmt and will squash it tomorrow morning

Reply:
The `AvailableToServe` is required in the sense when we create Backup record  at that time the RestAPI returns its value so that's why it could be empty.
The omitempty is written test make it fail safe